src/models/UDT/index.ts

```diff
@@ -17,3 +17,20 @@ export interface UDT {
   displayName?: string
   uan?: string
 }
+
+export enum MintStatus {
+  Minting = 'minting',
+  Closed = 'closed',
+  RebaseStart = 'rebase_start',
+}
+
+export interface OmigaInscription extends UDT {
```

Comment:
I would prefer naming it clearly to distinguish `inscription collection` and `inscription item`

Reply:
Indeed, it has been updated.